<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExceptionRecordForm</class>
 <widget class="QWidget" name="ExceptionRecordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QCheckBox,QLineEdit,QDateTimeEdit{
	font: 11pt &quot;Microsoft YaHei UI&quot;;
	color: rgb(255, 255, 255);
}
QLineEdit,QDateTimeEdit{	
	background-color: rgb(25, 32, 48);
}
QPushButton {
	font: 11pt &quot;Microsoft YaHei UI&quot;;
    color: white;
    background-color: rgba(0, 128, 187, 150);
    border-radius: 5px;
    border: 1px solid #0080bb;
    padding: 5px;
}
QPushButton:hover {
    background-color: rgba(0, 255, 255, 200);
    color: black;
}
QPushButton:pressed {
    background-color: rgba(0, 200, 200, 255);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>开始时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dtt_st">
       <property name="displayFormat">
        <string>yyyy/MM/dd HH:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>结束时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dtt_et">
       <property name="displayFormat">
        <string>yyyy/MM/dd HH:mm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>设备ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cbx_common">
       <property name="text">
        <string>一般异常</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbx_communication">
       <property name="text">
        <string>通讯异常</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbx_battery">
       <property name="text">
        <string>电池异常</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>搜索</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::SystemSearch"/>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_out_excel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_delete">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>一键删除</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::EditClear"/>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">QTableWidget
{
background-color: rgb(25, 32, 48);
color: rgb(255, 255, 255);
border: 1px solid #255990;
}
QHeaderView::section 
{
background-color: rgb(37, 89, 144);
color: white;
}</string>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
